server.js

```diff
@@ -1673,21 +1673,39 @@ cache(function(data, match, sendBadge, request) {
 }));
 
 // npm license integration.
-camp.route(/^\/npm\/l\/(.*)\.(svg|png|gif|jpg|json)$/,
+camp.route(/^\/npm\/l\/(@[^\/]*)?\/?([^\/]*)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Or better yet:

```
/^\/npm\/l\/(?:@([^\/]+)\/)?([^\/]+)\.(svg|png|gif|jpg|json)$/
```

This way, you can change the code below to:

```js
const pkg = encodeURIComponent(scope + '/' + repo);
apiUrl = `https://registry.npmjs.org/@${pkg}`;
```

By doing so, you would correctly escape both the scope and the package name